src/components/layouts/centered.js

```diff
@@ -4,7 +4,7 @@ import BaseLayout from './base'
 const CenteredLayout = (props) => {
   const style = {
     textAlign: props.left ? 'left' : 'center',
-    maxWidth: props.wide ? '900px' : '600px'
```

Comment:
@joostdecock I remember this fixes the width somewhere. I do not completely know the differences between the max-width for pages, but in my opinion, it should be responsive instead of hardcoded per page. What are your thoughts?

Reply:
Yes, of course things should be responsive. But this is not really about that. This essentially turns the *centered* layout into two different varieties. 

 - By default, you get a centered layout, that will responsively adapt but will not grow larger than 600px. The [login page](https://freesewing.org/login/) is an example of that layout in use.
 - If you pass the `wide` prop to this layout component, it will be the exact same thing, except now, it will be allowed to grow as wide as 900px. The [patterns page](https://freesewing.org/patterns/) is an example of a page using that layout.

It's sort of a middle ground where the wide layout would be too wide, and the centered layout too narrow.

As such, I think we should keep it. Because without a `max-width` the centered layout wouldn't be centered. It would just fill up the entire available space.

Reply:
Cheers!